benchmark/test/reference/spmv_distributed.profile.stderr

```diff
@@ -52,6 +52,10 @@ DEBUG: begin components::fill_array
 DEBUG: end   components::fill_array
 DEBUG: begin components::fill_array
 DEBUG: end   components::fill_array
+DEBUG: begin components::fill_array
+DEBUG: end   components::fill_array
+DEBUG: begin components::fill_array
+DEBUG: end   components::fill_array
```

Comment:
where is the fill_array from?

Reply:
Since this happens after the partition was created and before the device matrix data is created, I can only think of it being related to creating the index map in the matrix constructor. 
The index map contains two segmented arrays, each of which has an offsets array. This array always contains at least one element, so this is most likely from that.